boards/arm/tdk_robokit1/doc/index.rst

```diff
@@ -0,0 +1,155 @@
+
+.. image:: img/tdk_robokit1.jpg
+     :width: 500px
+     :align: center
+     :alt: TDK RoboKit1
+
+Hardware
+********
+
+- ATSAME70Q21 ARM Cortex-M7 Processor
+- 12 MHz crystal oscillator (Pres)
+- 32.768 kHz crystal oscillator
+- Micro-AB USB device
+- Micro-AB USB debug (Microchip EDBG) interface supporting CMSIS-DAP, Virtual COM Port and Data
+- JTAG interface connector
+- One reset pushbutton
+- One red user LED
+- TDK ICM 42688-P 6-Axis 32KHz IMU
+- TDK ICP-10111 Pressure Sensor
+- TDK NTC Thermistor for Temperature
+- AKM AK09918C Magnetometer
+- 2 TDK HVCi-4223 Cortex-M3 Dedicated Motor Controller
+- 3 TDK ICS-43434 Stereo Microphones
+- Connector for Industrial Dual IMU (TDK IIM-46230)
+- TDK CH101 Ultrasonic Range Sensor Array (9 Connectors, comes with 3)
+
+Supported Features
+==================
+
+The TDK RoboKit1 board supports the following hardware
+features:
+
++-----------+------------+-------------------------------------+
+| Interface | Controller | Driver/Component                    |
++===========+============+=====================================+
+| NVIC      | on-chip    | nested vector interrupt controller  |
++-----------+------------+-------------------------------------+
+| SYSTICK   | on-chip    | systick                             |
++-----------+------------+-------------------------------------+
+| UART      | on-chip    | serial port                         |
++-----------+------------+-------------------------------------+
+| USART     | on-chip    | serial port                         |
++-----------+------------+-------------------------------------+
+| I2C       | on-chip    | i2c                                 |
++-----------+------------+-------------------------------------+
+| I2S       | on-chip    | i2s                                 |
++-----------+------------+-------------------------------------+
+| SPI       | on-chip    | spi                                 |
++-----------+------------+-------------------------------------+
+| WATCHDOG  | on-chip    | watchdog                            |
++-----------+------------+-------------------------------------+
+| GPIO      | on-chip    | gpio                                |
++-----------+------------+-------------------------------------+
+| ADC       | on-chip    | ADC via AFEC                        |
++-----------+------------+-------------------------------------+
+| USB       | on-chip    | USB device                          |
++-----------+------------+-------------------------------------+
+| PWM       | on-chip    | pwm                                 |
++-----------+------------+-------------------------------------+
+| CAN       | on-chip    | canbus                              |
++-----------+------------+-------------------------------------+
+| HWINFO    | on-chip    | Unique device serial number         |
++-----------+------------+-------------------------------------+
```

Comment:
Let's move to new table entry format. You can use below as reference: https://github.com/zephyrproject-rtos/zephyr/blob/6ede1da43494da3840b2f270de955f6bf11cbbf0/boards/arm/gd32f450z_eval/doc/index.rst#L54-L86